.github/ISSUE_TEMPLATE.md

```diff
@@ -2,6 +2,13 @@
 Please, check https://docs.docker.com/docker-for-mac/troubleshoot/.
 Issues without logs and details cannot be debugged, and will be closed.
 
+Issues unrelated to Docker for Mac will closed.  In particular, see
+  - https://github.com/docker/compose/issues for docker-compose
+  - https://github.com/docker/machine/issues for docker-machine
+-->
```

Comment:
moby/moby for docker daemon issues ? (pas toujours facile pour le user de savoir)